src/api/index.ts

```diff
@@ -1,11 +1,16 @@
+// @ts-nocheck disable type-checking for now. turn me back on when feeling brave.
```

Comment:
i converted this file to ts in name only so that we can begin using ts in it. when the `@ts-nocheck` tag is at the top of the file, errors & warnings are suppressed. this is us tell ts "we know there's a bunch 'wrong' in this file". it will be a gradual process of cleaning it up as we go.